src/profiles_management/helpers/kfam.py

```diff
@@ -50,6 +62,99 @@ def resource_is_for_profile_owner(resource: GenericNamespacedResource | RoleBind
     return False
 
 
+def get_contributor_user(resource: GenericNamespacedResource | RoleBinding) -> str:
+    """Return user in KFAM annotation.
+
+    Raises:
+        ValueError: If the object does not have KFAM annotations.
+
+    Returns:
+        The user defined in metadata.annotations.user of the resource.
+    """
+    if not has_valid_kfam_annotations(resource):
+        raise ValueError("Resource doesn't have KFAM metadata: %s" % k8s.get_name(resource))
+
+    annotations = k8s.get_annotations(resource)
+    return annotations["user"]
+
+
+def get_contributor_role(
+    resource: GenericNamespacedResource | RoleBinding,
+) -> ContributorRole:
+    """Return role in KFAM annotation.
+
+    Raises:
+        ValueError: If the object does not have valid KFAM annotations.
+
+    Returns:
+        The user defined in metadata.annotations.user of the resource.
+    """
+    if not has_valid_kfam_annotations(resource):
+        raise ValueError("Resource doesn't have KFAM metadata: %s" % k8s.get_name(resource))
+
+    annotations = k8s.get_annotations(resource)
+    return ContributorRole(annotations["role"])
+
+
+def resource_matches_profile_contributor(
+    resource: RoleBinding | GenericNamespacedResource, profile: Profile
+) -> bool:
+    """Check if the user and it's role in the RoleBinding match the PMR.
```

Comment:
suggestion: grammar fix
```suggestion
    """Check if the user and its role in the RoleBinding match the PMR.
```

Reply:
I believe this is still valid.

Reply:
I think this one is resolved in https://github.com/canonical/github-profiles-automator/pull/36/commits/2cbd0aa88d87f76c6eb71f13b45d7b80b09d655d, but maybe I did this in another place 😅 

Taking a deeper look

Reply:
Did the following 2 greps and things look OK
```bash
grep " its " -ri
grep " it's " -ri
```